src/syrupy/__init__.py

```diff
@@ -63,6 +64,11 @@ def pytest_sessionstart(session: Any) -> None:
         warn_unused_snapshots=config.option.warn_unused_snapshots,
         update_snapshots=config.option.update_snapshots,
         base_dir=config.rootdir,
+        targeted_items={
+            arg
+            for arg in config.invocation_params.args
+            if not arg.startswith("-") and os.path.isfile(arg)
```

Comment:
can you run pytest with a partial filename like a prefix?

Reply:
It looks like you can run it with globby patterns like `tests/test_in*`. I'll rejig this to see if `glob` matches anything with the given pattern, that should cover all bases.